odl/set/space.py

```diff
@@ -753,6 +753,13 @@ def multiply(self, x, y):
         """
         return self.space.multiply(x, y, out=self)
 
+    def divide(self, x, y):
+        """Divide by ``other`` inplace.
```

Comment:
See #248. For now at least the doc should be fixed, like

    Implement ``self[:] = x * y``.